src/main/resources/application-local.properties

```diff
@@ -1,29 +1,31 @@
 server.port=8809
-spring.datasource.url={PG_DATABASE}/postgres
-spring.datasource.url=jdbc:postgresql://${PGHOST}:${PGPORT}/${PG_DATABASE}
-spring.datasource.username=${PG_RW_USER}
-spring.datasource.password=${PG_RW_PASSWORD}
+
+spring.datasource.url=jdbc:postgresql://localhost:5431/sync_db
+spring.datasource.username=sync_user
+spring.datasource.password=sync_password
 spring.datasource.platform=staging
 spring.jpa.properties.hibernate.default_schema=staging
 spring.jpa.show-sql=true
 
 logging.level.org.springframework.security=INFO
 
-lrsservice.url=http://elrr-external-services:8080
+lrsservice.url=http://localhost:8088
 lrsservice.cookie=
 
 cronExpression=0 0/1 * * * *
 
 lrs.samlurl=https://mocksaml.com/api/saml/metadata
 lrs.samlid=samltest
 
-brokerUrl=elrr-kafka:9092
+brokerUrl=localhost:9092
+
 kafka.topic=test-1
 kafka.groupId=test-consumer-group
+kafka.partitions=1
+kafka.replicas=1
 
 spring.http.multipart.enabled=true
 spring.http.multipart.max-file-size=2MB
 spring.http.multipart.max-request-size=2MB
 
-server.use-forward-headers=true
-
+##server.use-forward-headers=true
```

Comment:
your IDE is killing EOF lines, which github is not a fan of. Make sure to leave extra last line if it was there initially.

Reply:
not addressed